regression/goto-analyzer-simplify/simplify-complex-expression/test.desc

```diff
@@ -0,0 +1,5 @@
+CORE
+main.c
+"--variable"
```

Comment:
The command line should be task / abstract interpreter / domain (and possibly output).